x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/enrich/LookupFromIndexServiceResponseTests.java

```diff
@@ -48,12 +49,27 @@ LookupFromIndexService.LookupResponse createTestInstance(BlockFactory blockFacto
      * here.
      */
     Page testPage(BlockFactory blockFactory) {
```

Comment:
Now we can rightfully call this `randomPage`, haha.

Let's update the javadoc on top, though, that's out of date.

Reply:
:+1: